worlds/Files.py

```diff
@@ -184,6 +185,7 @@ def get_manifest(self) -> Dict[str, Any]:
             "player": self.player,
             "player_name": self.player_name,
             "game": self.game,
+            "patch_file_ending": self.patch_file_ending,
```

Comment:
What's the usefulness of putting this information in the manifest?
This seems like a duplication of information. What happens, and what should happen if the file has a different ending from what's in the manifest?